aws_embedded_metrics/environment/environment_detector.py

```diff
@@ -32,6 +37,19 @@ async def resolve_environment() -> Environment:
         log.debug("Environment resolved from cache.")
         return EnvironmentCache.environment
 
+    print(repr(Config.environment))
+    if Config.environment:
+        if Config.environment.lower() == "lambda":
+            EnvironmentCache.environment = lambda_environment
+        elif Config.environment.lower() == "ec2":
```

Comment:
we're calling `.lower()` multiple times which will result in unnecessary string allocation. Can we just do this once at the top?

Reply:
changed -- good call